libs/components/layout/testing/src/fluid-grid/fluid-grid-harness.ts

```diff
@@ -0,0 +1,62 @@
+import { HarnessPredicate } from '@angular/cdk/testing';
+import { SkyComponentHarness } from '@skyux/core/testing';
+
+import { SkyFluidGridHarnessFilters } from './fluid-grid-harness-filters';
+import { SkyRowHarness } from './row-harness';
+
+/**
+ * Harness for interacting with a fluid grid component in tests.
+ */
+export class SkyFluidGridHarness extends SkyComponentHarness {
+  /**
+   * @internal
+   */
+  public static hostSelector = 'sky-fluid-grid';
+
+  #getGrid = this.locatorFor('.sky-fluid-grid');
+  #getRows = this.locatorForAll(SkyRowHarness);
+
+  /**
+   * Gets a `HarnessPredicate` that can be used to search for a
+   * `SkyFluidGridHarness` that meets certain criteria
+   */
+  public static with(
+    filters: SkyFluidGridHarnessFilters,
+  ): HarnessPredicate<SkyFluidGridHarness> {
+    return SkyFluidGridHarness.getDataSkyIdPredicate(filters);
+  }
+
+  /**
+   * Gets the gutter size for the grid.
+   */
+  public async getGutterSize(): Promise<string> {
+    const grid = await this.#getGrid();
+
+    const small = await grid.hasClass('sky-fluid-grid-gutter-size-small');
+    const medium = await grid.hasClass('sky-fluid-grid-gutter-size-medium');
+
+    if (small) {
+      return 'small';
+    } else if (medium) {
+      return 'medium';
+    } else {
+      return 'large';
+    }
+  }
+
+  /**
+   * Gets all of the rows in the grid.
+   */
+  public async getRows(): Promise<SkyRowHarness[]> {
+    return await this.#getRows();
+  }
+
+  /**
+   * Whether the fluid grid has additional margins on the sides.
+   */
+  public async hasMargin(): Promise<boolean> {
```

Comment:
is this linked to the `disableMargin` input? if so can we update the jsDocs to say something like 
```Whether the fluid grid has margin enabled.```

Reply:
updated